### PR TITLE
fix: add Adit0507

### LINKED ISSUE #1185: Aditya Kumar Singh
## Profile Screenshot

Capture

### PR DESCRIPTION
**Pull request by Adit0507**
resolves #1185 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Adit0507.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/78037846/174435896-610e56c8-ecec-479b-b572-014a7873de6b.PNG",
+    "issueId": 1185,
+    "name": "Aditya Kumar Singh",
+    "username": "Adit0507"
+  }
+
```
Comment (Panquesito7) [inline profiles/Adit0507.json:7]: Wrong indentation and missing an endline. This should fix it. 🙂
```suggestion
  "image": "https://user-images.githubusercontent.com/78037846/174435896-610e56c8-ecec-479b-b572-014a7873de6b.PNG",
  "issueId": 1185,
  "name": "Aditya Kumar Singh",
  "username": "Adit0507"
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Adit0507** [comment]
I just did that, is it working fine now ?

File: profiles/Adit0507.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+"image": "https://user-images.githubusercontent.com/78037846/174435896-610e56c8-ecec-479b-b572-014a7873de6b.PNG",
+"issueId": 1185,
+"name": "Aditya Kumar Singh",
+"username": "Adit0507"
```
Comment (schmelto) [inline profiles/Adit0507.json:5]:
```suggestion
  "image": "https://user-images.githubusercontent.com/78037846/174435896-610e56c8-ecec-479b-b572-014a7873de6b.PNG",
  "issueId": 1185,
  "name": "Aditya Kumar Singh",
  "username": "Adit0507"
```

**Comment by Adit0507** [comment]
I hope the indentation is fine now

**Pull request merged**